services/QuillLMS/app/helpers/public_pages_helper.rb

```diff
@@ -15,7 +15,8 @@ module PublicPagesHelper
 
   EVIDENCE_HANDBOOK_URL = "https://docsend.com/view/29vcfdqa5aupkmfp"
 
-  def should_render_react_component
+  def render_react_component?(current_user)
+    (current_user)
```

Comment:
I'm not clear on what this line is doing.

Reply:
Me either.  Good catch.